### PR TITLE
🩹 MakeFile various tweaks

### PR DESCRIPTION
**Pull request by Kiirx**
- IL2CPU was using Release config which broke the setup, so now every project will use Debug config.
- Added build time
- Added some coloring for easier reading because of the huge amount of warnings

image

### CODE REVIEW

**Comment by Kiirx** [comment]
it seems like the github actions aren't using the makefile, which is probably why the config problem passed through the test, the github action should be moved to a linux one instead to test both of the platforms

File: Makefile [hunk 1]
```diff
@@ -17,38 +17,47 @@ GIT = git
 DOTNET = dotnet
 
 GITFLAGS = clone --depth=1
-DOTNETFLAGS = -v:q -nologo
+DOTNETFLAGS = -nologo -v:q -c:Debug
```
Comment (zarlo) [inline Makefile:20]: i dont think we use debug on the windows install @valentinbreiz do you know?

Comment (Kiirx) [reply Makefile:20]: This is for Linux anyways

Comment (zarlo) [reply Makefile:20]: yes but we should do the same as windows when building

Comment (zarlo) [reply Makefile:20]: for build something with cosmos yes but if the windows build uses release to build cosmos we should do that on linux too

Comment (Kiirx) [reply Makefile:20]: We could make a setup for that, but for everything to be working for sure we will have to add this makefile to the github build action

**Comment by Kiirx** [comment]
so?

File: Makefile (hunk 1 above)
Comment (quajak) [reply Makefile:20]: I think we use release mode on windows as well. We only use debug when compiling the kernel using cosmos

**Review by valentinbreiz — CHANGES_REQUESTED** [review]
Good work @Kiirx but could you enable release mode again?

**Comment by Kiirx** [comment]
@valentinbreiz could you please review this PR? As it is ready to merge

**Review by valentinbreiz — APPROVED** [review]

**Pull request merged**